src/bin/pgcopydb/dump_restore.c

```diff
@@ -155,12 +157,27 @@ copydb_target_prepare_schema(CopyDataSpec *specs)
 		}
 	}
 
+	/* if restoring specific schemas as specified in the inclusion filter
+		make sure they exist in the target database, if not create them.
+	 */
+	 if (specs->filters.includeOnlySchemaList.count > 0)
+	 {
+		if (!copydb_target_prepare_namespaces(specs))
+		{
+			/* errors have already been logged */
+			return false;
+		}
+	 }
```

Comment:
I thought we would take care of that automatically within the pg_dump and pg_restore calls. If that's not happening, I think the comment ought to say why and give some context. I would prefer that we don't have to create the schemas here, that said, because what if the user made a typo in their filtering setup? Then we create unwanted schemas on the target databases...

Also nitpicking: Please follow the code style in use all around in the same code/file. It looks like this:

``` c
	/* 
     * If restoring specific schemas as specified in the inclusion filter
	 *	make sure they exist in the target database, if not create them.
	 */
```

Note the capital, empty lines, and stars at the beginning of all lines within the comment.